src/components/manage/Blocks/Teaser/schema.js

```diff
@@ -96,8 +100,19 @@ export const TeaserSchema = ({ intl }) => {
     actions: ['left', 'right', 'center'],
     default: 'left',
   };
+  schema.properties.styles.schema.properties.scale = {
+    title: intl.formatMessage(messages.scale),
+    choices: [
+      ['mini', 'Mini'],
+      ['preview', 'Preview'],
+      ['teaser', 'Teaser'],
+      ['large', 'Large'],
+      ['great', 'Great'],
+    ],
```

Comment:
The scale names should be passed through intl to get them translated.

For future reuse of the scale list, perhaps this list should be configurable in the block configuration, this way a developer can change the list of the available scales without overriding the component. 

Moreover, right now we do not have an API endpoint to expose the available image scales, but in the future I think that the list of available scales should come from the API endpoint call.

Reply:
Thanks for the review. I will keep this in mind for the future contributions.